eva-submission-docker/postgres_docker/contig_alias.sql

```diff
@@ -0,0 +1,109 @@
+CREATE USER contig_alias_user WITH PASSWORD 'contig_alias_pass';
+CREATE DATABASE contig_alias;
+-- Connect to the database
+\c contig_alias
+
+CREATE ROLE eva;
+
+CREATE SCHEMA eva AUTHORIZATION contig_alias_user;
+
+CREATE SEQUENCE eva.hibernate_sequence
+	INCREMENT BY 1
+	MINVALUE 1
+	MAXVALUE 9223372036854775807
+	START 1
+	CACHE 1
+	NO CYCLE;
+
+ALTER SEQUENCE eva.hibernate_sequence OWNER TO contig_alias_user;
+GRANT ALL ON SEQUENCE eva.hibernate_sequence TO contig_alias_user;
+
+CREATE TABLE eva.assembly (
+	insdc_accession varchar(255) NOT NULL,
+	is_genbank_refseq_identical bool NOT NULL,
+	md5checksum varchar(255) NULL,
+	"name" varchar(255) NOT NULL,
+	organism varchar(255) NULL,
+	refseq varchar(255) NULL,
+	taxid int8 NOT NULL,
+	trunc512checksum varchar(255) NULL,
+	genbank varchar(255) NULL,
+	CONSTRAINT assembly_pkey PRIMARY KEY (insdc_accession)
+);
+
+ALTER TABLE eva.assembly OWNER TO contig_alias_user;
+GRANT ALL ON TABLE eva.assembly TO contig_alias_user;
+
+CREATE TABLE eva.chromosome_b4_eva2964_schema_change (
```

Comment:
Can we remove these `b4_eva2964` tables?  Hopefully contig-alias doesn't need them...